### ISSUE TITLE
bug: OpenAI Integration (Python): Incompatibility with openai python library v1

### ISSUE THREAD

**Issue by janik-sauerbier** [langfuse/langfuse#450]
### Describe the bug

I tried to use the [OpenAI Integration (Python)](https://langfuse.com/docs/integrations/openai), but I got the following error. Maybe I did something wrong here, but the same error appears in the Google Colab.

```
ModuleNotFoundError                       Traceback (most recent call last)
[<ipython-input-6-3d72670053ac>](https://localhost:8080/#) in <cell line: 2>()
      1 # instead of: import openai
----> 2 from langfuse.openai import openai

[/usr/local/lib/python3.10/dist-packages/langfuse/openai.py](https://localhost:8080/#) in <module>
      4 
      5 import openai
----> 6 from openai.api_resources import ChatCompletion, Completion
      7 
      8 from langfuse import Langfuse

ModuleNotFoundError: No module named 'openai.api_resources'
```

### To reproduce

I just followed the basic setup. You can see it in the Google Colab. Maybe my local setup is just rigged somehow.

### Additional information

Might be connected to the name of the file "openai.py" [See here](https://githubmemory.com/repo/openai/openai-python/issues/3)

**Comment by gyliu513** [comment]
Did you run the following command?

```
pip install langfuse openai --upgrade
```

**Comment by maxdeichmann** [comment]
> Did you run the following command?
> 
> ```
> pip install langfuse openai --upgrade
> ```

Thanks, @gyliu513 for jumping in here.

**Comment by janik-sauerbier** [comment]
@gyliu513 Thanks! I tried that and followed the instructions, but I'm getting the same error locally and in the Google Colab: https://colab.research.google.com/github/langfuse/langfuse-docs/blob/main/cookbook/integration_openai_sdk.ipynb

**Comment by maxdeichmann** [comment]
@janik-sauerbier yes, agree. We will look into compatibility with 1.0.0 as soon as possible.

**Comment by maxdeichmann** [comment]
Hi all, this was fixed in the Python SDK version 1.2.1 https://github.com/langfuse/langfuse-python/releases/tag/v1.2.1

**Comment by maxdeichmann** [comment]
Hi all, this was fixed in the Python SDK version 1.2.1 https://github.com/langfuse/langfuse-python/releases/tag/v1.2.1

**Comment by janik-sauerbier** [comment]
Thanks a lot! :)
